### PR TITLE
Made mathicsscript XDG compatible

### PR DESCRIPTION
**Pull request by GarkGarcia**
Made mathicsscript [XDG](https://wiki.archlinux.org/index.php/XDG_Base_Directory)-compatible.

### CODE REVIEW

File: __main__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+/home/pablo/Documents/mathicsscript/mathicsscript/__main__.py
```
Comment (rocky) [inline __main__.py:1]: Was this added inadvertently?

**Review by rocky — COMMENTED** [review]
Other than the very specific path, this looks fine - Thanks!

File: __main__.py (hunk 1 above)
Comment (GarkGarcia) [reply __main__.py:1]: Yes, I didn't intent to add this (I don't event know what it is).

**Review by rocky — APPROVED** [review]
LGTM

**Comment by rocky** [comment]
@GarkGarcia The new HISTFILE location was not working for me so I reverted that change in 815cd34

**Comment by GarkGarcia** [comment]
> @GarkGarcia The new HISTFILE location was not working for me so I reverted that change in [815cd34](https://github.com/Mathics3/mathicsscript/commit/815cd343ffeb38221da4a2aec0b59f2a7e50dfb0)

I've noticed it didn't work for me too. I don't really know why (changing the file's path shouldn't break anything), but I'll try to investigate it in the following weeks.

**Pull request merged**